core/config/config.go

```diff
@@ -34,12 +34,7 @@ func InitConfig(configPath string) error {
 	}
 	// First Sentinel will try to load config from the given file.
 	// If the path is empty (not set), Sentinel will use the default config.
-	err := loadFromYamlFile(configPath)
-	if err != nil {
-		return err
-	}
-
-	return OverrideConfigFromEnvAndInitLog()
+	return loadFromYamlFile(configPath)
```

Comment:
`loadFromYamlFile` does more than one things, 1) load config from yaml file; 2) set  `globalCfg` . May be better to refactor  it.